### DISCUSSION TITLE
Is it possible to restore a development database from a backup

### DISCUSSION THREAD

**Discussion by markns** [Tecnativa/doodba-copier-template#486] [General]
Hey, thanks for Doodba, it's been really helpful so far!

I have a couple of backup related questions:

1. Is it possible to use the `backup` service to create a `devel` database from the `prod` db for the purposes of building a realistic dev env? Or is the duplicity backup not intended for this purpose.
2. We will ultimately run postgres on AWS RDS, rather than in the PG docker container. Is duplicity suitable for RDS too?

Thanks :)

**Comment by pleirb** [comment]
> Gracias por Doodba, ha sido muy útil hasta ahora.
> 
> Tengo un par de preguntas relacionadas con la copia de seguridad:
> 
>     1. Es posible utilizar el  `backup`servicio para crear un  `devel`base de datos de la  `prod`db para la construcción de un dev realista env? O es la copia de seguridad de duplicidad no destinada a este propósito.
> 
>     2. En última instancia, correremos postgres en AWS RDS, en lugar de en el contenedor de atador PG. Es la duplicidad adecuada para RDS también?
> 
> 
> Gracias:)

To use a database in development mode, I have found it helpful to do this:
**Option 1 – Restore a backup.dump**
`docker-compose run -l traefik.enable=false --rm -v /home/user/patch/to/bkp:/bkp odoo pg_restore -U odoo -v -d prod /bkp/backup.dump`
**Option 2 – Restore a backup.sql**
`docker compose run -l traefik.enable=false --user odoo --rm -v /home/user/patch/to/bkp:/bkp odoo bash -c 'psql -d prod < /bkp/backup.sql'`
**And to copy the filestore**
```
docker compose up -d
docker compose cp /home/user/patch/to/filestore/. odoo:/var/lib/odoo/filestore/prod
```
